File: Php/app/Urls/PlayerDeleteUrlModel.php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\PlayerUrlModel;
	use entities\Player;
	use entities\Homegame;

	class PlayerDeleteUrlModel extends PlayerUrlModel{

		public function __construct(Homegame $homegame, Player $player){
			parent::__construct(RouteFormats::playerDelete, $homegame, $player);
		}

	}

}

File: Web/Models/UrlModels/PlayerDeleteUrlModel.cs
using Core.Classes;
using Web.Routing;

namespace Web.Models.UrlModels{
    public class PlayerDeleteUrlModel : PlayerUrlModel{

		public PlayerDeleteUrlModel(Homegame homegame, Player player) : base(RouteFormats.PlayerDelete, homegame, player){
		}

	}

}